examples/wrapper-types/uniffi.toml

```diff
@@ -9,3 +9,30 @@ cdylib_name = "wrapper_types"
 
 [bindings.kotlin]
 cdylib_name = "wrapper_types"
+
+# Python wrapper implementation for JsonObject
+[wrappers.JsonObject.python]
+
+code = """
```

Comment:
In the last proposal, I separated out the Python/Kotlin code into their own files and I still think that has some merit.  Here's how I'm thinking about that:
 - We can start by just using a string in the `.toml file`
 - If we decide we need something like that, then maybe we could add an way for `uniffi.toml` to include other files.  Maybe instead of including the code inline you could do something like `code = "@include("uniffi/json-object.py")`

Reply:
IMO, one of the most important things here is that each consumer can specify a different uniffi.toml than the component itself - which I think would naturally lead to allowing each language binding to have its own too.

I don't think the use-case for including .py code makes much sense TBH - uniffi doesn't need to get involved here - the consumer could just stick `json-object.py` wherever they like and just import it. It might make sense for other languages though, and maybe I'm missing your point a little.